examples/local/README.md

```diff
@@ -16,6 +17,11 @@ guide, all placeholders must be replaced with sensible values.
 - *COMMON_DOMAIN* - Cluster's etcd and API common domain.
 - *COMMON_DOMAIN_INGRESS* - Ingress common domain.
 - *AZURE_LOCATION* - Azure location.
+- *AZURE_CLIENT_ID* - Client ID for the Active Directory Service Principal.
+- *AZURE_CLIENT_SECRET* - Client Secret for the AD Service Principal.
```

Comment:
Once you say `Active Directory` and once `AD`. Is this on purpose? Do you mean the same or is `AD` something else?

Reply:
I meant the same. Changed for consistency.